shared/html/options.html

```diff
@@ -31,31 +31,20 @@ <h1 class="ddg-logo"><span>DuckDuckGo.com</span></h1>
 
   <section class="divider-bottom">
 
-    <p class="options-info">
-      Search and browse the web without being tracked.
-    </p>
+    <p class="options-info" id="options-sub-header"></p>
```

Comment:
Mhm, here I'm concerned that we are removing any hint of content from the markup. What if instead we used a `data` attribute and used it instead of the `id` to target content? Something like:

```html
<p class="options-info" data-text="options:optionsSubHeader"></p>
```

Then you can do:

```js
const textContainers = document.querySelectorAll('[data-text]');
textContainers.forEach(el => {
    const textID = el.getAttribute('data-text');
    const text = t(textID)
    el.innerHTML = text;
});
```

I think it's more maintainable and leaves traces of the text content within the markup.

Reply:
Done!